src/Assets/Pack/Microsoft.AspNetCore.Assets.Pack.csproj

```diff
@@ -0,0 +1,201 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <Description>ASP.NET Core static framework assets</Description>
+    <PackageId>Microsoft.AspNetCore.Assets.Pack</PackageId>
+    <IsPackable>true</IsPackable>
+    <OutputType>Library</OutputType>
```

Comment:
Why `library`?

Reply:
Why not? This is not producing any exe/dll

Reply:
Wanted to make sure it's intentional, I don't know off the top of my head if this controls any behavior we care about

Reply:
No, it's conceptually a library